docs/docs/sample-scripts/spo/manage-news-page-types/index.md

```diff
@@ -0,0 +1,45 @@
+---
+tags:
+  - customizations
+  - libraries
+---
+
+# Create custom views to differentiate SharePoint news page types in Site Pages library
+
+Author: [João Ferreira](https://sharepoint.handsontek.net/2020/08/23/effectively-manage-sharepoint-news-part-1/)
```

Comment:
You should put your name here 🙂. You can always say that you were inspired by this blog.

Author: [Nanddeep Nachan](https://github.com/nanddeepn), Inspired by [João Ferreira](https://sharepoint.handsontek.net/2020/08/23/effectively-manage-sharepoint-news-part-1/)